apps/gas-estimate/README.md

```diff
@@ -0,0 +1,5 @@
+# Gas Estimate Function
+Gas Estimate function provides a simple endpoint to estimate Ethereum mainnet Gas price. It is expected to be used as a 
+nodejs lambda function. It requires to include an environment variable, `process.env.ETH_GAS_STATION`, which is an Etherscan API Key.
```

Comment:
I'd add `AWS` to refer that lambda is an AWS product, because this might not be clear to some people.
I'd add too a ref to `https://api.etherscan.io` (e.g. Etherscan (https://api.etherscan.io) API Key).